get_hashsets.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const AppConstants = require("./app-constants").init();
+
+const fs = require("fs");
+const path = require("path");
+const request = require("request");
+
+const pkg = require("./package.json");
+
+const HIBP_AUTH = 'Bearer ' + new Buffer(AppConstants.HIBP_API_TOKEN);
+
+
+function getBreachHashset(breach) {
+  if (breach.IsActive && breach.IsVerified && breach.DataClasses.includes("Email addresses")) {
+    console.log("Active, verified breach with email addresses: " + breach.Name);
+    const url = AppConstants.HIBP_API_ROOT + '/enterprisesubscriber/hashset/' + breach.Name;
+    console.log("url: " + url);
+
+    const headers = {
+      'User-Agent': `${pkg.name}@${pkg.version}`,
+      'Authorization': HIBP_AUTH
+    }
+    const hashsetRequestObject = {
+      url: url,
+      headers: headers
+    };
+
+    request(hashsetRequestObject).pipe(fs.createWriteStream('breach_hashsets' + path.sep + breach.Name + '.zip'));
+  }
+}
+
+function handleBreachesResponse(error, response, body) {
+  let breachesJSON = JSON.parse(body);
+  for (let breach of breachesJSON) {
+    getBreachHashset(breach);
+  }
+}
+
+const breachesRequestObject = {
+  url: AppConstants.HIBP_API_ROOT + '/breaches',
+  headers: {
+    'User-Agent': 'request'
+  }
+}
```

Comment:
nit: missing semicolon

Reply:
Beautiful, another eslint opportunity has been revealed

Reply:
Wait, the `semi` rule should be taking care of this, never mind.

Reply:
ESLint _was_ catching this. The opportunity for CI has been revealed (and there should already be an open PR for it).